src/call_log/call_log.c

```diff
@@ -217,4 +217,34 @@ __wt_call_log_open_session(WT_SESSION_IMPL *session, int ret_val)
     return (0);
 }
 
+/*
+ * __wt_call_log_query_timestamp --
+ *     Print the call log entry for the query timestamp API call.
+ */
+int
+__wt_call_log_query_timestamp(
+  WT_SESSION_IMPL *session, const char *config, const char *hex_timestamp, int ret_val)
+{
+    WT_CONNECTION_IMPL *conn;
+    char config_buf[128], hex_ts_buf[128], objectid_buf[128];
+
+    conn = S2C(session);
+
+    WT_RET(__call_log_print_start(session, "connection", "query_timestamp"));
+
+    /* Input arguments for the API call query_timestamp. */
+    WT_RET(__wt_snprintf(config_buf, sizeof(config_buf), "\"config\": \"%s\"", config));
+    WT_RET(__call_log_print_input(session, 1, config_buf));
+
+    /* Output arguments for the API call query_timestamp. */
+    WT_RET(__wt_snprintf(objectid_buf, sizeof(objectid_buf), "\"object_id\": \"%p\"", conn));
```

Comment:
You can change this now to reflect WT-9215

Reply:
That is to move the object id outside of the output array.

Reply:
Yup on it now.

Reply:
Updated in [081d141](https://github.com/wiredtiger/wiredtiger/pull/8095/commits/081d141a9fc378c9cf68b8a059012692ce84bcbe)